scripts/covid_tracking_project/historic_state_data/preprocess_csv.py

```diff
@@ -0,0 +1,60 @@
+import csv
```

Comment:
(later is fine) might be good if the example has a test. that way, there is some chance that contributors will provide one.